File: Npc/11001632.cs
using Maple2.Model.Enum;
using Maple2.Script.Npc;

/// <summary>
/// 11001632: Eupheria
/// </summary>
public class _11001632 : NpcScript {
    protected override int First() {
        return 10;
    }

    // Select 0:
    // $script:0517100407006134$
    // - Sniff... 
    protected override int Select() => 0;

    protected override int Execute(int selection) {
        switch (Id, Index++) {
            case (10, 0):
                // $script:0517100407006135$
                // - This is all my doing... 
                return -1;
        }

        return default;
    }

    protected override NpcTalkButton Button() {
        return (Id, Index) switch {
            (10, 0) => NpcTalkButton.Close,
            _ => NpcTalkButton.None,
        };
    }
}

File: Npc/11001632.py
""" 11001632: Eupheria """
from npc_api import Option, Script
import random


class Main(Script):
    def first(self) -> int:
        return 10

    def select(self) -> int:
        return 0

    def __0(self, pick: int) -> int:
        # $script:0517100407006134$
        # - Sniff... 
        return None # TODO

    def __10(self, pick: int) -> int:
        # $script:0517100407006135$
        # - This is all my doing... 
        return -1

    def button(self) -> Option:
        if (self.state, self.index) == (10, 0):
            return Option.CLOSE
        return Option.NONE
